Tasks/UseDotNetV2/versionfetcher.ts

```diff
@@ -294,4 +293,4 @@ export class DotNetCoreVersionFetcher {
     }
 }
 
-const DotNetCoreReleasesIndexUrl: string = "https://raw.githubusercontent.com/dotnet/core/master/release-notes/releases-index.json";
+const DotNetCoreReleasesIndexUrl: string = "https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/releases-index.json";
```

Comment:
should we also make this change in the DotNetCoreInstaller task?

Reply:
minor: this should also be updated in task.json

Reply:
Done